docs/userGuide/components/searchbar.md

```diff
@@ -34,10 +34,15 @@ To use the searchbar within a navbar, add the following markup to your file. The
 
 Name | Type | Default | Description
 ---- | ---- | ------- | ------
-data | `Array` || The local data source for suggestions. Expected to be a primitive array.
+data | `Array` || The local data source for suggestions. Expected to be a primitive array. To use MarkBind's search functionality, set this value to `"searchData"`.
 menu-align-right | `Boolean` | `false` | Whether the search bar's dropdown list will be right-aligned.
 on-hit | `Function` || A callback function when you click or hit return on an item.
 placeholder | `String` | `''` | The placeholder text shown when no keywords are entered in the search bar.
 
+<box type="warning">
+
+Warning: If `data` is set to `"searchData"`, then `enableSearch` **must be set to `true` in `site.json`**. See: [Configuring the Site: enableSearch]({{ baseUrl }}/userGuide/siteConfiguration.html#enable-search).
```

Comment:
In our documentation, we use `Note:` instead of `Warning: (within `Site Configuration` section)

Adding a line break before "See: ...." makes it cleaner to read as well.